### PR TITLE
Make the changePage and scrollTo functionality accesable to the public

### PR DESCRIPTION
**Pull request by tschob**
Following changes are made to provide the ability to scroll to anchors from outside of the FolioReaderKit:
- make the `FolioReaderPage` class public
- Add a public getter for the `FolioReaderCenter` to the `FolioReaderContainer`
- Add a public getter for the current `FolioReaderPage` to the `FolioReaderCenter`

### CODE REVIEW

File: Source/FolioReaderCenter.swift [hunk 1]
```diff
@@ -772,6 +761,23 @@ public class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UICo
         }
         return nil
     }
+
+	// MARK: Public page methods
+
+	public func changePageWith(page page: Int, animated: Bool = false, completion: (() -> Void)? = nil) {
```
Comment (hebertialmeida) [inline Source/FolioReaderCenter.swift:774]: Please add some docs for public functions, so you help me on generated docs 😀

File: Source/FolioReaderContainer.swift [hunk 2]
```diff
@@ -188,4 +188,11 @@ public class FolioReaderContainer: UIViewController {
     override public func preferredStatusBarStyle() -> UIStatusBarStyle {
         return isNight(.LightContent, .Default)
     }
+
+	// MARK: - Public
+
+	public func readerCenter() -> FolioReaderCenter {
```
Comment (hebertialmeida) [inline Source/FolioReaderContainer.swift]: Why do we need this? it is already exposed thought singleton.

You can use it:

``` swift
let readerCenter = FolioReader.sharedInstance.readerCenter
```

File: Source/FolioReaderCenter.swift [hunk 3]
```diff
@@ -772,6 +761,23 @@ public class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UICo
...
+			let indexPath = NSIndexPath(forRow: page-1, inSection: 0)
+			changePageWith(indexPath: indexPath, animated: animated, completion: { () -> Void in
+				self.updateCurrentPage({ () -> Void in
+					completion?()
+				})
+			})
+		}
+	}
+
+	public func getCurrentPage() -> FolioReaderPage? {
```
Comment (hebertialmeida) [inline Source/FolioReaderCenter.swift]: I think is better to make `currentPage` public get with a private set, so we don't need it.

``` swift
public private(set) var currentPage: FolioReaderPage?
```

And use it

``` swift
FolioReader.sharedInstance.readerCenter.currentPage
```

What do you think?

File: Source/FolioReaderContainer.swift (hunk 2 above)
Comment (tschob) [reply Source/FolioReaderContainer.swift]: Oh, didn't see this one. 
In general the new method would support an non singleton usage. But as there might be many places in the FolioReader which doesn't support a usage with multiple instances this doesn't matter at the moment.

File: Source/FolioReaderCenter.swift (hunk 3 above)
Comment (tschob) [reply Source/FolioReaderCenter.swift]: It's fine for me.

**Comment by tschob** [comment]
@hebertialmeida I have update my PR based on your comments.

**Pull request merged**